maintenance/wikia/Forum/fixCommentIndexes.php

```diff
@@ -0,0 +1,308 @@
+<?php
+/**
+ * fixCommentIndexes
+ *
+ * Jira: https://wikia-inc.atlassian.net/browse/SOC-1485
+ *
+ * This script fixes the issue where the same comment index number is used on multiple comments, e.g. the #2 in
+ * the fragment portion of this URL:
+ *
+ *   http://creepypasta.wikia.com/wiki/Thread:510920#2
+ *
+ * For example, the URLs for 5 comments on a thread might be broken in this way:
+ *
+ *   http://creepypasta.wikia.com/wiki/Thread:510920#2
+ *   http://creepypasta.wikia.com/wiki/Thread:510920#3
+ *   http://creepypasta.wikia.com/wiki/Thread:510920#4
+ *   http://creepypasta.wikia.com/wiki/Thread:510920#4
+ *   http://creepypasta.wikia.com/wiki/Thread:510920#4
+ *
+ * where the same index, #4, has been used for three comments.  The fix would be to update the index for each of
+ * these comments (stored in the page_wikia_props table) to be unique, e.g.:
+ *
+ *   http://creepypasta.wikia.com/wiki/Thread:510920#2
+ *                                               ...#3
+ *                                               ...#4
+ *                                               ...#5
+ *                                               ...#6
+ *
+ * Its possible that this streak of duplicates can happen within normally numbered comments.  This is because
+ * we released a fix for the problem before creating this script.  All we need to do is make sure we only update
```

Comment:
This bug was caused by PLATFORM-1658, right? Can we add that ticket # here for some additional context about what that fix was?